vaadin-core/pom.xml

```diff
@@ -40,7 +40,7 @@
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
-            <artifactId>vaadin-dev-server</artifactId>
+            <artifactId>vaadin-dev</artifactId>
```

Comment:
With this modules structure, we would need to remove `vaadin` from `vaadin-dev-bundle` (which is now set as an optional) and replace it with the entire list of components dependencies. Do you think it would be enough to add an optional scope to all of the component in this list to avoid leaking commertial component if the dev bundle dep is added? Asking because this would be a next step and good to figure out the structure in adnvance :)

Reply:
No, I mean adding optional to the dependencies in the vaadin-dev-bundle.

Reply:
this structure looks good to me.. did i miss something?